templates/author.twig

```diff
@@ -39,5 +39,17 @@
 				</ul>
 			</div>
 		</div>
+
+		{% if posts.pagination.total > 1 %}
+			<div class="row">
+				<div class="col-lg-8">
+					<button
+						data-page="1"
```

Comment:
I'm using data attributes here but we could use a different method, for example, calling a JS function using twig variables as in:

```
<script>
  setupAuthorLazyLoad('{{ author.path }}', '{{ posts.pagination.total }}')
</script>
```

For now, I think the `data` method is fine.

Reply:
I think it is good for now also! :+1: